pkg/webhook/kapiserver/ensurer.go

```diff
@@ -163,3 +162,58 @@ func ensureKubeAPIServerIsMutated(ps *corev1.PodSpec, c *corev1.Container, caBun
 		},
 	})
 }
+
+// getLatestIssuedCABundleSecret returns the oidc-webhook latest CA bundle secret
+func getLatestIssuedCABundleSecret(ctx context.Context, c client.Client, namespace string) (*corev1.Secret, error) {
```

Comment:
@rfranzke Can you please review the logic in this function? Just to confirm that this is the desired way of getting the newest ca bundle. Everything else looks good to me.